core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.spec.ts

```diff
@@ -67,6 +69,22 @@ describe('OperatorLabelComponent', () => {
     expect(jqueryElement.data('uiDraggable')).toBeTruthy();
   });
 
+  it('should display operator description shortly after hovering on a operator label', () => {
+    const spy = spyOn<any>(component, 'mouseEnter');
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    operatorLabelElement.triggerEventHandler('mouseenter', component);
+    expect(spy).toHaveBeenCalled();
+    expect(mockOperatorData.additionalMetadata.operatorDescription).toContain(operatorLabelElement.attributes['ng-reflect-ngb-tooltip']);
+  });
+
+  it('should hide operator descritption once the cursor leaves a operator label', () => {
+    const spy = spyOn<any>(component, 'mouseLeave');
+    const operatorLabelElement = fixture.debugElement.query(By.css('#' + component.operatorLabelID));
+    operatorLabelElement.triggerEventHandler('mouseleave', component.tooltipWindow);
+    expect(spy).toHaveBeenCalled();
```

Comment:
Same thing from above, use marble to trigger the event. For this test, you need to first trigger the `mouseEnter` event. Then, trigger `mouseLeave` event and check the event triggered time and changes in` operatorLabelElement`.